app/main.py

```diff
@@ -1 +1,46 @@
-#  write your code here
+class Animal:
+    animals = []
+
+    def __init__(self, name: str,
+                 appetite: int, is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry is True:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        return 0
+
+
+class Cat(Animal):
+    def __init__(self, name: str, appetite: int = 3,
+                 is_hungry: bool = True) -> None:
+        super().__init__(name, appetite, is_hungry)
+
+    @staticmethod
+    def catch_mouse() -> None:
+        print("The hunt began!")
+
+
+class Dog(Animal):
+    def __init__(self, name: str, appetite: int = 7,
+                 is_hungry: bool = True) -> None:
+        super().__init__(name, appetite, is_hungry)
+
+    @staticmethod
+    def bring_slippers() -> None:
+        print("The slippers delivered!")
+
+
+def feed_animals(animals_list: list) -> int:
+    sum_points = 0
+    for animal in animals_list:
+        if animal.feed():
```

Comment:
use `feed` method to get number of the appetite points

Reply:
What do you mean? In 44 string I use feed method to get number of the appetite each animal